code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -26,94 +35,96 @@
 
 	//exosuits and helmets obscure our view and stuff.
 	if(wear_suit)
-		skipsuitstorage |= wear_suit.flags_inv & HIDESUITSTORAGE
+		if(wear_suit.flags_inv & HIDESUITSTORAGE)
+			skip_gear |= EXAMINE_SKIPSUITSTORAGE
+
 		if(wear_suit.flags_inv & HIDEJUMPSUIT)
-			skiparms |= 1
-			skiplegs |= 1
-			skipchest |= 1
-			skipgroin |= 1
-			skipjumpsuit |= 1
-			skiptie |= 1
-			skipholster |= 1
+			skip_body |= EXAMINE_SKIPARMS | EXAMINE_SKIPLEGS | EXAMINE_SKIPBODY | EXAMINE_SKIPGROIN
+			skip_gear |= EXAMINE_SKIPJUMPSUIT | EXAMINE_SKIPTIE | EXAMINE_SKIPHOLSTER

```

Comment:
What if instead you made EXAMINE_SKIPJUMPSUIT just the sum of all these bits rather than a single bit? Is there ever a time where you want to skip the jumpsuit but also not skip the tie and holster?